### PR TITLE
Small fixes, fix inheritance

### PR DESCRIPTION
**Pull request by potapovDim**
## Motivation/Description of the PR
- Fix inheritance

## Type of change
- [ x] :nail_care: Polish code

## Checklist:
- [ x] Lint checking (Run `npm run lint`)
- [ x] Local tests are passed (Run `npm test`)

### CODE REVIEW

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ lib/command/run.js  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=5195954&bid=17040531)

**Comment by potapovDim** [comment]
@DavertMik Hey, should i remove this PR?

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ lib/command/run.js  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=5195954&bid=17040531)

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- lib/helper/WebDriver.js  3
- lib/helper/WebDriverIO.js  1
         

Clones removed
==============
+ test/runner/dry_run_test.js  -1
+ test/helper/Puppeteer_test.js  -1
+ test/helper/Playwright_test.js  -1
+ lib/command/run.js  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=5195954&bid=17040531)

File: lib/codecept.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ const resolve = require('path').resolve;
 
 const container = require('./container');
 const Config = require('./config');
-const event = require('../lib/event');
```
Comment (Vorobeyko) [inline lib/codecept.js]: Make rebase by master, please.

File: lib/command/configMigrate.js [hunk 2]
```diff
@@ -8,8 +8,8 @@ const util = require('util');
 const print = require('../output').print;
 const success = require('../output').success;
 const error = require('../output').error;
-const fileExists = require('../utils').fileExists;
-const getTestRoot = require('./utils').getTestRoot;
+const { fileExists } = require('../utils');
```
Comment (Vorobeyko) [inline lib/command/configMigrate.js]: `const { fileExists, getTestRoot } = require('../utils');`

File: lib/command/run-multiple/collection.js [hunk 3]
```diff
@@ -138,8 +138,8 @@ class Collection {
    * If value of property `browser` does not match the slected `browser` in conjugation
    * with the selectedRunName then the suite is removed from configuration.
    */
-  filterSelectedBrowsers(selectedRuns, config) {
-    selectedRuns.forEach((selectedRun) => {
+  filterSelectedBrowsers(selectedRuns) {
+    selectedRuns.forEach(() => {
       selectedRuns.forEach((selectedRun) => {
```
Comment (Vorobeyko) [inline lib/command/run-multiple/collection.js]: We really need to the same loop on loop? =)

File: lib/command/workers/runTests.js [hunk 4]
```diff
@@ -1,23 +1,20 @@
 const tty = require('tty');
 
-if (!tty.getWindowSize) {
+if ('getWindowSize' in tty) {
```
Comment (Vorobeyko) [inline lib/command/workers/runTests.js]: `!tty.getWindowSize` this syntax usually using.

File: lib/command/workers/runTests.js [hunk 5]
```diff
@@ -1,23 +1,20 @@
...
   // this is really old method, long removed from Node, but Mocha
   // reporters fall back on it if they cannot use `process.stdout.getWindowSize`
   // we need to polyfill it.
-  tty.getWindowSize = () => [40, 80];
+  Object.defineProperty(tty, 'getWindowSize', { value: () => ([40, 80]) });
```
Comment (Vorobeyko) [inline lib/command/workers/runTests.js]: Why?

File: lib/command/workers/runTests.js [hunk 6]
```diff
@@ -59,7 +56,7 @@ function filterTests() {
 function initializeListeners() {
   function simplifyTest(test, err = null) {
     if (test.start && !test.duration) {
-      const end = new Date();
+      const end = +new Date();
```
Comment (Vorobeyko) [inline lib/command/workers/runTests.js]: why?

Comment (potapovDim) [reply lib/command/workers/runTests.js]: one useless

File: lib/command/workers/runTests.js (hunk 4 above)
Comment (potapovDim) [reply lib/command/workers/runTests.js]: should i back this to initial condition, assertion and initialization of "getWindowSize" ?

File: lib/command/configMigrate.js (hunk 2 above)
Comment (potapovDim) [reply lib/command/configMigrate.js]: ./utils and ../utils they are not the same

File: lib/command/run-multiple/collection.js (hunk 3 above)
Comment (potapovDim) [reply lib/command/run-multiple/collection.js]: i did not implement on logic of this function

File: lib/command/workers/runTests.js (hunk 5 above)
Comment (potapovDim) [reply lib/command/workers/runTests.js]: will back

File: lib/command/workers/runTests.js (hunk 4 above)
Comment (potapovDim) [reply lib/command/workers/runTests.js]: will back

File: lib/codecept.js (hunk 1 above)
Comment (potapovDim) [reply lib/codecept.js]: i do not have conflicts for rebase

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- lib/helper/WebDriverIO.js  1
- lib/helper/WebDriver.js  3
         

Clones removed
==============
+ test/helper/Puppeteer_test.js  -1
+ test/runner/dry_run_test.js  -1
+ lib/command/run.js  -1
+ test/helper/Playwright_test.js  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=5195954&bid=17040531)

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- lib/helper/WebDriverIO.js  1
- lib/helper/WebDriver.js  3
         

Clones removed
==============
+ lib/command/run.js  -1
+ test/helper/Playwright_test.js  -1
+ test/runner/dry_run_test.js  -1
+ test/helper/Puppeteer_test.js  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=5195954&bid=17040531)

**Pull request merged**